### ISSUE TITLE
Hotkey for clickthrough

### ISSUE THREAD

**Issue by gitsamhub** [mobzystems/SeeThroughWindows#2]
Thank you for great utility.
Could you please make it possible to assign hotkey for 'Click through' function?
Even better if hotkey acts as a toggle function. So that I can navigate both windows flawlessly.
Thank you.

**Comment by mobzystems** [comment]
Good idea. I don't see the UI for it yet but we can think o something ;-)
